### PR TITLE
Deploy Arb-Sepolia - Remove Arb-Goerli

### PR DESCRIPTION
**Pull request by codenamejason**

### CODE REVIEW

File: scripts/test/createProfile.ts [hunk 1]
```diff
@@ -13,7 +12,7 @@ export async function createProfile() {
 
     const balance = await ethers.provider.getBalance(deployerAddress);
 
-    const registry = registryConfig[chainId].registryProxy;
+    const registry = "0x4AAcca72145e1dF2aeC137E1f3C5E3D75DB8b5f3";
```
Comment (thelostone-mc) [inline scripts/test/createProfile.ts:15]: ^ can we change this back to fetch it from the config ?

File: .openzeppelin/unknown-421614.json [hunk 2]
```diff
@@ -0,0 +1,515 @@
+{
```
Comment (thelostone-mc) [inline .openzeppelin/unknown-421614.json:1]: we can remove the arb goerli as it's not used

File: scripts/config/allo.config.ts [hunk 3]
```diff
@@ -95,10 +95,10 @@ export const alloConfig: DeployParams = {
...
     baseFee: 0,
   },
-  // Arbitrum Goerli
-  421613: {
-    owner: "0xE64c2171bB7c7412eb2ad3c4c027C499FEf345DC",
-    treasury: "0xE64c2171bB7c7412eb2ad3c4c027C499FEf345DC",
+  // Arbitrum Sepolia
+  421614: {
+    owner: "0x8C180840fcBb90CE8464B4eCd12ab0f840c6647C",
+    treasury: "0x8C180840fcBb90CE8464B4eCd12ab0f840c6647C",
```
Comment (thelostone-mc) [inline scripts/config/allo.config.ts:101]: Is this a reason we use a different EOA than we what had used on Arb Goeri ?

**Review by thelostone-mc — CHANGES_REQUESTED** [review]
Left in a few comments but otherwise LGTM

File: scripts/test/createProfile.ts (hunk 1 above)
Comment (codenamejason) [reply scripts/test/createProfile.ts:15]: This doesn't exist on the config currently. We never updated the script when we updated the configs.

Two Options:
1. Just hardcode when we test creating a profile and pool after deploy on testnets.
2. Add the `registryProxy` to the config on all networks (has to be added to the type).

File: .openzeppelin/unknown-421614.json (hunk 2 above)
Comment (codenamejason) [reply .openzeppelin/unknown-421614.json:1]: I am curious to why the `arbitrum-sepolia.json` file is not present 🤔

File: scripts/config/allo.config.ts (hunk 3 above)
Comment (codenamejason) [reply scripts/config/allo.config.ts:101]: There were a few different ones so we used the one with funds and was already present in the code.

**Review by thelostone-mc — APPROVED** [review]

**Pull request merged**